code/modules/research/stock_parts.dm

```diff
@@ -199,6 +205,13 @@ If you create T5+ please take a pass at gene_modder.dm [L40]. Max_values MUST fi
 	rating = 2
 	custom_materials = list(/datum/material/iron=80)
 
+/obj/item/stock_parts/electrolite/adv
+	name = "advanced electrolite"
+	desc = "An electrolite component to separate matters into others"
+	icon_state = "adv_electrolite"
+	rating = 2
+	custom_materials = list(/datum/material/iron = 50, /datum/material/glass = 50)
```

Comment:
I think parts adv and highter must take adv materials, like another stock parts

Reply:
wym?

Reply:
I misunderstand something here.